.github/actions/verify-badge/action.yaml

```diff
@@ -0,0 +1,257 @@
+
+            # Check for built-in podMonitors
+            POD_MONITOR_COUNT=$(kubectl get PodMonitor -n "${NAMESPACE}" -o json | jq -r '.items | length')
+            if [ "$POD_MONITOR_COUNT" -eq 0 ]; then
+              echo "❌ No monitors defined"
+              exit 1
+            else
+              echo "✅ PodMonitor found"
+            fi
+          else
+            echo "✅ ServiceMonitor found"
+          fi
+        else
+          echo "✅ Monitor defined in UDS Package"
+        fi
+
+        echo
+
+        # Exemptions
+
+        ## Must minimize the scope and number of the exemptions to only what is absolutely required by the application
+
+        EXEMPTION_COUNT=$(kubectl get Exemptions -n "${NAMESPACE}" -o json | jq -r '.items | length')
+        if [ "$EXEMPTION_COUNT" -gt 0 ]; then
+          echo "::warning ::⚠️ Exemptions are present, review needed"
+        fi
+        echo "✅ No exemptions present"
+
+        echo
+
+        # Structure
+
+        ## Should expose all configuration (uds.dev CRs, additional Secrets/ConfigMaps, etc) through a Helm chart (ideally in a chart or charts directory).
+
+        COMMON_ZARF_MANIFEST_COUNT=$(cat common/zarf.yaml | yq '.components[] | select(.manifests != null) | length' | wc -l)
+        if [ "$COMMON_ZARF_MANIFEST_COUNT" -gt 0 ]; then
+          echo "❌ Manifests present in common/zarf.yaml"
+          exit 1
+        fi
+        echo "✅ No manifests present in common/zarf.yaml"
+
+        MAIN_ZARF_MANIFEST_COUNT=$(cat zarf.yaml | yq '.components[] | select(.manifests != null) | length' | wc -l)
+        if [ "$MAIN_ZARF_MANIFEST_COUNT" -gt 0 ]; then
+          echo "❌ Manifests present in zarf.yaml"
+          exit 1
+        fi
+        echo "✅ No manifests present in zarf.yaml"
+        ## Should implement or allow for multiple flavors (ideally with common definitions in a common directory)
+
+        ZARF_COMPONENTS_FLAVOR_COUNT=$(cat zarf.yaml | yq '.components[] | select(.only.flavor != null) | length' | wc -l)
+        if [ "$ZARF_COMPONENTS_FLAVOR_COUNT" -eq 0 ]; then
+          echo "❌ No flavors defined in zarf.yaml"
+          exit 1
+        fi
+        echo "✅ At least one flavor defined in zarf.yaml"
+
+        echo
+
+        # Testing
+
+        ## Must implement Journey Testing to cover the basic user flows and features of the application, especially where an application interacts with an external service / interface.
+
+        if [ -d "tests" ]; then
+          if [ "$(ls -A tests)" ]; then
+            echo "✅ Tests folder exists and has files"
+          else
+            echo "❌ Tests folder exists but is empty"
+            exit 1
+          fi
+        else
+          echo "❌ Tests folder does not exist"
+          exit 1
+        fi
+
+        echo
+
+        ## Should lint their configurations with appropriate tooling such as yamllint and zarf dev lint.
+
+        pre-commit run -a
```

Comment:
we're not using pre-commit and had actually discussed removing the configs that were left over from copy/paste from older repos. If im not mistaken this would also require installing pre-commit, which isnt happening. IMO this should be removed. 

You could check for lint steps in the tasks and workflows instead of running pre-commit

Reply:
Removed pre-commit, linting is already covered elsewhere in CI so I think it's fine